ansible-tests/tripleo-ansible-inventory.py

```diff
@@ -93,6 +94,7 @@ def __init__(self, configs):
         self._ksclient = None
         self._hclient = None
         self._nclient = None
+	self.stack_name = self.get_stack_name()
```

Comment:
This is not indented correctly. It should be 8 spaces, but it seems to be a tab plus a space instead. I'm actually surprised Python doesn't complain.